### PR TITLE
feat(data model): adds related evidence to task model property

### PR DESCRIPTION
**Pull request by eve-horne**
# Changes introduced
Updated the data model to have support for the evidence required <-> task model relationship
Added handling to the import script for this data model change

## Related issue(s):
- CSE-280

## Contributor checklist

- [x] Wrote/updated tests for introduced changes
- [ ] ~~Added new stories for created/modified components (if applicable)~~
- [ ] ~~Updated Postman library for created/modified routes (if applicable)~~
- [x] Verified that there are no issues in CircleCI
- [x] Assigned yourself to the PR
- [x] Removed `WIP:` from the PR title
- [x] Requested review from the SB Reviewers team

## Reviewer checklist

- [x] Assigned yourself to the PR
- [x] Read through the changes in the `Files changed` tab
- [x] Verified that tests were written/modified
- [ ] ~~Verified that stories were written/modified (if applicable)~~
- [ ] ~~Verified that Postman was updated (if applicable)~~
- [x] Checked out the branch and tested changes locally

### CODE REVIEW

File: src/models/target/index.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
 // tslint:disable:no-reserved-keywords
```
Comment (puregarlic) [inline src/models/target/index.ts]: This file needs to be formatted before it's merged. Otherwise, it looks great.

**Review by puregarlic — CHANGES_REQUESTED** [review]

File: src/models/target/index.ts [hunk 2]
```diff
@@ -23,16 +22,21 @@ export interface IStandards {
   stdDesc: string;
 }
 
+export interface IEvidence {
```
Comment (danvanhorn) [inline src/models/target/index.ts:25]: Good thinking adding an interface for this. we may have to update the types on the client side too so the client-side team can take advantage of them.

**Review by danvanhorn — APPROVED** [review]
I've ran everything locally and tested the changes. Good job.

**Pull request merged**